src/graphviz.h

```diff
@@ -30,12 +30,16 @@ struct GraphViz {
   GraphViz(State* state, DiskInterface* disk_interface)
       : dyndep_loader_(state, disk_interface) {}
   void Start();
-  void AddTarget(Node* node);
+  void AddTarget(Node* node, int depth);
   void Finish();
 
+  void printNodeLabel(Node* node);
+
   DyndepLoader dyndep_loader_;
-  std::set<Node*> visited_nodes_;
+  std::set<Node*> visited_nodes_;  
```

Comment:
Please remove all trailing spaces from your commit (i.e. squash your second one into the first).